modules/rtp_rtcp/source/rtcp_receiver.cc

```diff
@@ -945,6 +950,42 @@ RtcpStatisticsCallback* RTCPReceiver::GetRtcpStatisticsCallback() {
   return stats_callback_;
 }
 
+#ifdef INTEL_TELEMETRY
+// Currently it's a simplified decision making process:
+// if nack lists grows to larger than 20, we have a burst loss;
+// and otherwise, if we have consecutive 4 packets that is missing,
+// it is also a burst loss.
+webrtc::PacketLossPattern RTCPReceiver::GetCurrentPacketLossPattern(
+  const std::vector<uint16_t>& nack_sequence_numbers) {
+  if (nack_sequence_numbers.size() == 0)
+    return PacketLossPattern::kPacketLossNone;
+
+  if (nack_sequence_numbers.size() >= 20)
```

Comment:
An optional suggestion: define a variable for the threshold of burst loss, so it can be easily changed.